src/commands/information/info.ts

```diff
@@ -61,13 +61,7 @@ class InfoCommand extends Command {
             }
         }
 
-        let robloxMember: GroupMember;
-        try {
-            robloxMember = await robloxGroup.getMember(robloxUser.id);
-            if(!robloxMember) throw new Error();
-        } catch (err) {
-            return ctx.reply({ embeds: [ getRobloxUserIsNotMemberEmbed() ]});
-        }
+        let robloxMember: GroupMember = await robloxGroup.getMember(robloxUser.id);
```

Comment:
Why did you change this? The previous system had necessary error handling.

Reply:
Due to group banning kicking the user, and it not allowing the checking of someone that's not in the group made doing this impossible without removing the member check, but don't worry, if a user isn't in a group, I just state that they're a Guest at rank 0 instead of letting it crash the bot